consensus/polybft/governance_manager.go

```diff
@@ -0,0 +1,541 @@
+package polybft
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/chain"
```

Comment:
Wanna order imports in the following order: built-in, third-party, internal?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1519/commits/4a92ab07bf4ef242b2a664bf3fa2c71509a7289d